src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -4172,13 +4172,12 @@
 """;
         var comp = CreateCompilation(src, new[] { comp2.EmitToImageReference() }, targetFramework: TargetFramework.Net70);
         comp.VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=13&length=17)

It doesn't look like there is a meaningful change in this statement.

Reply:
During iterations, I'd touched this test (pasted updated diagnostics). The new one differs by a space "... referenced. You ..." which is correct, so I'll leave.